serial_protocol.md

```diff
@@ -0,0 +1,188 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “BEGIN” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
```

Comment:
nit: since `$` is already used in the protocol, maybe we should using a different delimeter for placeholder like `%VALUE%`, `[VALUE]` or `<VALUE>`